models/movie.js

```diff
@@ -1,16 +1,14 @@
-const mongoose = require("mongoose");
+import mongoose from "mongoose";
 
 const movieSchema = mongoose.Schema({
-  _id: mongoose.Schema.Types.ObjectId,
   title: { type: String, required: true },
-  numberInStock: { type: Number, required: true },
```

Comment:
why is `numberInStock` being removed?

Reply:
I couldn't find a specific reason for including the`numberInStock` in this app. If such a reason exists, please let me know

Reply:
Fair enough.